### ISSUE TITLE
Simple Invoke-Command script doesn't return data from remote PC correctly

### ISSUE THREAD

**Issue by illusive-man** [PowerShell/PowerShell#13412]
Pretty much what title says. When invoking from one workstation simple Get-ChildItem on remote PC, the Powershell 7.0.3 session  neither returns nor renders the content of given drive which is unexpected (see the screenshots provided below). Running the same script in Windows powershell 5.1 session works just fine. 

## Steps to reproduce
Two workstations in the same network. One is Win10 Pro x64 v1903 with PS 7.0.3, the second is on Windows 7 SP1 x64 with PS 2.0
Remoting is enabled and working. WSMan's TrustedHosts - all hosts allowed on both WS.
```powershell
Invoke-Command -ComputerName 192.168.74.153 -ScriptBlock { Get-ChildItem 'C:\' } -credential DocsUser
```

## Expected behavior

```none
Like when run on PS 5.1:
![2020-08-12 13_50_11-Window](https://user-images.githubusercontent.com/25516639/90009510-dc7bbc80-dca6-11ea-9da4-b0bb4d97bca5.png)

```

## Actual behavior

```none
Powershell 7.0.3:
![2020-08-12 13_49_59-Window](https://user-images.githubusercontent.com/25516639/90009575-f7e6c780-dca6-11ea-823b-033859d167e8.png)
```

## Environment data

```none

Windows 10 workstation:
Name                           Value
----                           -----
PSVersion                      7.0.3
PSEdition                      Core
GitCommitId                    7.0.3
OS                             Microsoft Windows 10.0.18363
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

Windows 7 workstation:
Name                           Value
----                           -----
CLRVersion                     2.0.50727.8669
BuildVersion                   6.1.7601.23403
PSVersion                      2.0
WSManStackVersion              2.0
PSCompatibleVersions           {1.0, 2.0}
SerializationVersion           1.1.0.1
PSRemotingProtocolVersion      2.1
```

**Comment by vexx32** [comment]
IIRC this was an issue with the codeproperties the formatter tries to use or something. I'm pretty sure this got fixed fairly recently. Can you test on the latest preview release to be sure, though? 🙂

**Comment by illusive-man** [comment]
The latest one I found is PowerShell 7.1.0 Preview 3. Alas, the issue still there for that version.

**Comment by illusive-man** [comment]
Alas, the issue is still there for 7.1.0 preview 5.

**Comment by illusive-man** [comment]
@vexx32 I think you're right about formatter, since when I added "`| Format-table`" after "`gci`" the returned output is OK. But I suppose it's still the issue, since it should work without any additional actions/formats.

**Comment by vexx32** [comment]
Ah, knew I'd seen it before. There's a bit more info in #11400. 

Reading back through that discussion, sounds like the most promising option would be to duplicate the format view definitions for `System.IO.FileInfo` and `System.IO.DirectoryInfo` and explicitly declare the same formatters for their deserialised versions to account for this.

I'll mark this as duplicate for now. If you're interested in looking at a fix yourself, I can point out the relevant code areas where the formatting is handled. If not, I might have a look in a while myself; it'd be good to get this fixed for 7.1. 🙂
